src/app/components/indicator/terra-indicator.component.ts

```diff
@@ -3,6 +3,22 @@ import {
     Input
 } from '@angular/core';
 
+export enum TerraIndicatorLabelTypeEnum
+{
+    DEFAULT = 'default',
+    PRIMARY = 'primary',
+    SUCCESS = 'success',
+    INFO = 'info',
+    WARNING = 'warning',
+    DANGER = 'danger',
```

Comment:
Die enum Einträge müssen klein bzw. in lowerCamelCase notiert werden (https://github.com/plentymarkets/terra/blob/stable7/CODING-CONVENTIONS.md#constants)